File: PHP Word Examples/ParagraphsApiDemo/getParagraphFormat.php
<?php

use \Spire\Cloud\Word\Sdk\Client\Api\ParagraphsApi;
use \Spire\Cloud\Word\Sdk\Client\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new ParagraphsApi($configuration);
$filename="GetParagraphFormat_1.docx";
$node_path = 'Section/0/Body/0';
$index_of_paragraph = 0;
$folder = "input";
$storage = null;
$password = null;
$result = $apiInstance->getParagraphFormat($filename, $node_path, $index_of_paragraph, $folder, $storage, $password);
print_r($result);

File: Python Word Examples/ParagraphsApiDemo/getParagraphFormat.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.ParagraphsApi(configuration)

file_name = 'GetParagraphFormat.docx'
node_path = 'Section/0/Body/0'
index = 1
extra = {'folder': "input"}
result = api.get_paragraph_format(file_name, node_path, index, **extra)
